app/controllers/orders_controller.rb

```diff
@@ -30,6 +28,12 @@ def create
     end
   end
 
+  def ship
+    order = Order.find(params[:id])
+    order.update!(status: "shipped")
+    redirect_to "/admin/dashboard"
```

Comment:
This 'ship' method is for an admin. When its clicked, the order is 'shipped' and then the user is no longer able to cancel it.